tests/microbenchmarks.py

```diff
@@ -68,6 +71,7 @@ def test_global_var_load(test_case):
             print(y)
     """), timed=True)
 
+
 def test_class_var_load(test_case):
     print("Running", "test_class_var_load")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils